frontend/src/metabase/app.js

```diff
@@ -22,7 +22,9 @@ import "metabase/plugins/builtin";
 
 // This is conditionally aliased in the webpack config.
 // If EE isn't enabled, it loads an empty file.
-import "ee-plugins"; // eslint-disable-line import/no-duplicates
+// eslint-disable-next-line import/order -- those files may have siede-effects, let's keep them as they are
+import { activateEEPlugins } from "ee-plugins"; // eslint-disable-line import/no-duplicates
+activateEEPlugins();
```

Comment:
I changed this from relying on the side effect, to manually call the function to activate the plugins